### DISCUSSION TITLE
Why does CP not find my heap

### DISCUSSION THREAD

**Discussion by nitroSkirata** [Courseplay/Courseplay_FS22#2540] [Q&A] [answered]
Hey,

I wanted to try out the heap unloader mode, but the Maus tells me it can't find my heap.

Can someone help my find a solution?
![image_2023-05-18_161545471](https://github.com/Courseplay/Courseplay_FS22/assets/9157835/4fec4c6a-5a6c-4dd3-9389-20f5b336de35)
![image_2023-05-18_161606057](https://github.com/Courseplay/Courseplay_FS22/assets/9157835/13bceb2c-c79b-4e7a-8e53-677b32a735ff)

PS: the combine unloader can find the heap and mark it on the map
![image_2023-05-18_162526080](https://github.com/Courseplay/Courseplay_FS22/assets/9157835/5254db2d-ce2e-4f94-a3de-599c3c4b16f0)

kind regards,
nitro

**Comment by Tensuko** [comment, marked as answer]
Did you set the heap position the same way as the combine unloader ?
I did a test like 2 hours ago and it works just fine.

**Reply by nitroSkirata** [reply]
Ok, i was just too inprecise with the position of the node, and tried to use the wrong node.

thought the scan would be in a wider area.

works fine now, thanks.
